test/camunda-cloud/ModelerSpec.js

```diff
@@ -143,4 +143,19 @@ describe('<CamundaCloudModeler>', function() {
 
   });
 
-});
\ No newline at end of file
+
+  it('should inject descriptionContextProvider', function() {
+
+    // when
+    return createModeler(simpleXml).then(function(result) {
+
+      var modeler = result.modeler,
+          propertiesPanel = modeler.get('propertiesPanel');
+
+      // then
+      expect(propertiesPanel._descriptionConfig).to.exist;
```

Comment:
Is there any way to test this without referring to private properties?

Reply:
afaik not directly. We have the [BpmnPropertiesPanelRenderer](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/render/BpmnPropertiesPanelRenderer.js) at hand here, which does not offer ways to get its config